lib/ansible/module_utils/redfish_utils.py

```diff
@@ -729,6 +758,15 @@ def set_bios_attributes(self, attr):
             return response
         return {'ret': True, 'changed': True, 'msg': "Modified BIOS attribute"}
 
+    def set_multi_bios_attributes(self, attr):
```

Comment:
This "multi" method isn't needed in this PR since this is for a "set" method called from redfish_config.

Reply:
Done